### ISSUE TITLE
OKViz Sparklines visual too slow block completely Power BI

### ISSUE THREAD

**Issue by big2smart** [okviz/Sparkline#15]
I have implemented a visualization in POWER BI using your visuals and it takes too much time to load the data, in addition it blocks every other action I would like to make in the dashboard. 
I am using this visual in several sheets but in particular one of them has critical problems, in which I have several lines (at least 20.000 rows) to display (products with monthly trend as sparklines using a lookup measure to switch 3 different kpi). I can't even select the sheet, because now I get a blank page and nothing loads anymore.

It seems that with several lines and several format/design changes (highest point red, lowest green and others black, labels displayed) the visual becomes too slow and has problems to load.

Please tell me how to increase speed and avoid this type of problem. 
Attached you find screenshots of the sheets and the problems I get.

Thank you for your time
regards
BigSmart

<img width="970" alt="sparklines_report-example" src="https://user-images.githubusercontent.com/42494876/44299995-02f8fe00-a300-11e8-8a7f-abe42de4c92c.png">
<img width="640" alt="sparklines_report-example-with-errors" src="https://user-images.githubusercontent.com/42494876/44300000-11471a00-a300-11e8-81af-ea9dc123d4b6.PNG">

**Comment by danieleperilli** [comment]
Please, send a repro (pbix file) to support@okviz.com. We will investigate according to our priority list. Thanks.

**Comment by big2smart** [comment]
Dear Mr. Perilli,

please find the file attached.
If you select “all” by customer_group filter, you will notice that the file becomes really slow and the visuals is completely blocked.

In addition isn’t possible to adjust and make the rows a little bit higher (see the names in the files are to compressed)? And even the columns? To decide how large they have to be? For the moment I only see how the category text shall be distributed (left or right).

Waiting for your feedback

Thank you very much
Best regards
Vanessa

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by danieleperilli** [comment]
Vanessa, no files are attached.

**Comment by big2smart** [comment]
Dear Daniele,

Do you see the attachment now?
Btw strange.. it was attached in the previous email,  see below:
[cid:image002.png@01D43EAB.D9BCB620]
Waiting for your feedback,

Best regards
Vanessa

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by danieleperilli** [comment]
No, sorry. Try to write to support@okviz.com, instead of replying to the GitHub notification.
